forem/forem
# Android Runtime Banner - Chrome Intent support

## What type of PR is this? (check all applicable)


## Description

Runtime Banner Android app support 

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

- Open the app using iOS -> browse site and click on banner -> Should deep link into mobile app
- Open the app using Android -> browse site and click on banner -> Should deep link into mobile app

### UI accessibility concerns?

None - UI hasn't changed

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: The Android mobile app will be communicated accordingly in sync with the Marketing team

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
